src/Data/Hours/HoursData.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lits\LibCal\Data\Hours;
+
+use Lits\LibCal\Data;
+
+class HoursData extends Data
+{
+    public ?int $lid = null;
+    public ?string $name = null;
+    public ?string $category = null;
+    public ?string $desc = null;
+
+    public ?string $url = null;
+    public ?string $contact = null;
+    public ?string $lat = null;
+    public ?string $lon = null;
+    public ?string $color = null;
+    public ?string $fn = null;
+
+    public string $parent_lid;
```

Comment:
This should be nullable, as it isn't provided in the response for a whole Library.